ecommerce/baskets/serializers.py

```diff
@@ -0,0 +1,10 @@
+from django.db.transaction import atomic
+from rest_framework import serializers
+
+from baskets.models import Basket
+
+class BasketSerializer(serializers.ModelSerializer):
+
```

Comment:
BasketDetailedSerializer eksik. Customer Basket'e FK olarak verildiği için bir nested serializer yapısı kullanılması iyi olur.